libraries/mainMethods.js

```diff
@@ -178,34 +178,94 @@ async function sendGamesReport(gamesroom, members) {
     }
 }
 
-function showLadder(playerData, guildID) {
-    const maxPlayers = 20;
+function showLadder(playerData, guildID, ladderType) {
+    const maxPlayersPerEmbed = 10;
+    const embedArrayLimit = 10;
     let addedPlayers = 0;
     let guildData = bot.guilds.cache.get(guildID);
-    var ladderName = guildData.name + ' - AOE4 Ladder';
-    var embedData = new EmbedBuilder()
-        .setColor('#0099ff')
-        .setTimestamp()
-        .setFooter({ text: 'AOE4 Companion', iconURL: 'https://i.imgur.com/AfFp7pu.png' })
-        .setTitle(ladderName);
+    let ladderName = guildData.name + ' ' + ladderType;
+    let embedsArray = [];
+
+    // for testing purposes
+    playerData = addGeneratedTestData(playerData);
```

Comment:
should this be here?

Reply:
honestly, it is not used, not needed for proper work of code just for dev purouses

Reply:
done